client/src/components/Header/HeaderBgColor.js

```diff
@@ -0,0 +1,22 @@
+// Dictionary of our routes and corresponding background colors. 
+// Accessed by headerBackgroundColor()
+const routeDict = {
+	'/' : 'rgba(0,0,0,0.2)',
+	'/login' : 'white'
+}
+
+// This function takes in a route and returns its background color
+// value located in routeDict.
+// Accessed by Header.js
+const headerBackgroundColor = (route) => {
```

Comment:
I would move this into the Header.js file. This doesn't need to be extracted out because it's so directly tied to the header.